web/src/main/java/se/inera/intyg/webcert/web/csintegration/integration/CSIntegrationService.java

```diff
@@ -285,4 +289,16 @@ public Certificate signCertificateWithoutSignature(SignCertificateWithoutSignatu
 
         return response.getCertificate();
     }
+
+    public IntygPdf printCertificate(String certificateId, PrintCertificateRequestDTO request) {
+        final var url = baseUrl + CERTIFICATE_ENDPOINT_URL + "/" + certificateId + "/pdf";
+
+        final var response = restTemplate.postForObject(url, request, PrintCertificateResponseDTO.class);
+
+        if (response == null) {
+            return null;
```

Comment:
Lets throw exception instead of returning null. Would be great if you could check and add that to any other methods not throwing.